orchestration/cleanup_completed_agents.py

```
@@ -92,16 +129,79 @@ def check_agent_completion(agent_name: str) -> Dict[str, Any]:
+                "timeout_seconds": timeout_seconds,
+                "elapsed_hours": elapsed_seconds / 3600
+            }
+        else:
+            return {
+                "timeout": False,
+                "reason": "within_timeout",
+                "elapsed_seconds": elapsed_seconds,
+                "timeout_seconds": timeout_seconds
+            }
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)